the_board_game_tracker/templates/games/boardgame_detail.html

```diff
@@ -1,10 +1,20 @@
 {% extends "base.html" %}
 
-{% load games_extras %}
+{% load games_extras static %}
```

Comment:
static no longer required
```suggestion
{% load games_extras %}
```